test/unit/cli/parse-options-test.js

```diff
@@ -68,6 +72,57 @@ test('parse options', function (group) {
     t.end()
   })
 
+  group.test('dbUrl', function (t) {
```

Comment:
I was just playing with test cases for the moment. I will leave them as a final job. I would like to finish with the code at first.